tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md

```diff
@@ -501,3 +504,26 @@ AccuracyChecker supports following set of adapters:
   * `reg_max` - maximal value of integral set (optional, default 7).
   * `strides` - strides of input multi-level feature maps (optional, default [8, 16, 32]).
   * `is_legacy` - using a legacy NanoDet model (optional, default False).
+* `palm_detection` - converting output of Palm detection models family to `DetectionPrediction` representation.
```

Comment:
is it really models family???
probably it is specific model from mediapipe (such comment will be more informative for person who want reuse your code)

Reply:
AFAIK part of mediapipe code corresponding to palm_detection adapter adapter functionality implemented  as a multiclass detection code. So palm_detection adapter and can be used for other mediapipe models.

Reply:
then maybe palm detection family is not good name, something like 
`converting output of Palm detection model from mediapipe` or make adapter is more generic/rename to call it as mediapipe multiclass detection or something similar (it is not obvious that adapter for palm can be used for something else

Reply:
Fixed